### ISSUE TITLE
Which license should we choose for editorial content like Landing or Blog?

### ISSUE THREAD

**Issue by dpordomingo** [src-d/guide#69]
The [gide:licensing](https://github.com/src-d/guide/blob/master/engineering/licensing.md) only defines:
- [GPL v3.0](https://www.gnu.org/licenses/gpl.html), applied to all projects whose nature is specific to core parts of our stack
- [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0), to projects which are libraries or tools that intended to be consumed by third-parties as a source code dependencies

But there are editorial content like Landing or Blog that could not fit under these descriptions.

Should be one of them, or should they be relicensed under a common  `Copyright (c) ${YEAR} source{d}` as done by [landing](https://github.com/src-d/landing/blob/master/LICENSE) or [blog](https://github.com/src-d/blog/blob/master/LICENSE)?

**Comment by vmarkovtsev** [comment]
Thanks @dpordomingo
BTW I suggest CC-BY-NC 3.0

**Comment by dpordomingo** [comment]
Why `no-comercial`? (With that license the contents could not be used in many sites, for example, our contents could **not** be used in Wikipedia)

**Comment by vmarkovtsev** [comment]
Wikipedia is a strong argument... I just wanted to restrict bad people from making money.

**Comment by dpordomingo** [comment]
Since we allow people making money with our code if they attribute to us the authorship of our libraries, I see that it could be the same with our editorial contents.

**Comment by jorgeschnura** [comment]
Why not use Apache 2.0? The fact that a case like this was not described in our original intention to for that license doesn't mean we can't adopt it
cc @eiso

**Comment by marnovo** [comment]
> Regarding the version of the CC -if we choose one of them- I'm not sure if it should be considered a v4.

Agreed, if we're to go with a Creative Commons license, v4.0 seems much improved over v3.0. Any thoughts on ShareAlike (CC-BY-SA)? Also [CC v4.0 seems compatible with GPLv3].(https://www.fsf.org/blogs/licensing/cc-by-4-0-and-cc-by-sa-4-0-added-to-our-list-of-free-licenses)

> Why not use Apache 2.0? The fact that a case like this was not described in our original intention to for that license doesn't mean we can't adopt it

Anyone can comment on this? AFAIK Apache and the alike are not usual for "content", but I'm far from a licensing expert…

**Comment by vmarkovtsev** [comment]
I found this:
https://wiki.creativecommons.org/wiki/ShareAlike_compatibility_analysis:_GPL
https://www.quora.com/Does-it-make-sense-to-license-text-documentation-under-the-Apache-Software-License-instead-of-Creative-Commons
TL;DR: the subject of a software license is the source which is different from the CC's. Besides, Cc is closer to the domain details.

**Comment by eiso** [comment]
Was this decided and if so, is it reflected in the guide?

**Comment by jorgeschnura** [comment]
@dpordomingo @vmarkovtsev @marnovo ping

**Comment by dpordomingo** [comment]
Sorry, I feel I have no enough expertise to decide it, but I'd consider:
- a license close to the domain: editorial content -> CC as @vmarkovtsev wrote https://github.com/src-d/guide/issues/69#issuecomment-348481909
- a license compatible with Wikipedia -> allowing commercial use
- a license compatible with GPLv3 as said by @marnovo https://github.com/src-d/guide/issues/69#issuecomment-348468855

**Comment by campoy** [comment]
According to [this guy](https://twitter.com/talideon/status/950893555689623552) Wikipedia is dually licensed as CC BY-SA and [GFDL](https://en.wikipedia.org/wiki/GNU_Free_Documentation_License).

I still think CC-BY-SA is probably the best option at this point.

**Comment by dpordomingo** [comment]
GFDL seems to be [incompatible](https://en.wikipedia.org/wiki/GNU_Free_Documentation_License#GPL_incompatible_in_both_directions) (in both directions) with GPLv.3 (which we're already using in our core stuff)

**Comment by eiso** [comment]
To me CC-BY-SA makes sense.

**Comment by marnovo** [comment]
Alright, given everything I'm convinced we should go with CC-BY-SA.
@mcuadros also agreed with it IRL.

**Comment by eiso** [comment]
Closing this issue. It's settled.

**Comment by eiso** [comment]
Wait, this was PR'd right @marnovo?

**Comment by marnovo** [comment]
I don't see any so I will PR it to https://github.com/src-d/guide/blob/master/engineering/licensing.md

**Comment by marnovo** [comment]
PR for review on https://github.com/src-d/guide/pull/106.
